db/migrate/20211220185935_clean_duplicate_content_units.rb

```diff
@@ -0,0 +1,144 @@
+class CleanDuplicateContentUnits < ActiveRecord::Migration[6.0]
+  def find_duplicate_groups(model, fields)
+    model.having("count(*) > 1").group(*fields).pluck("string_agg(id::TEXT, ',') ").map do |ids|
```

Comment:
Interesting. Not seen string_agg before. Guessing it groups ids separated by comma.